### PR TITLE
Feature/danh vt/t01 show the total number of jobs

### PR DESCRIPTION
**Pull request by vutiendanhvgu01**

### CODE REVIEW

**Comment by tiencmavixo** [comment]
Please, check message commit

File: src/containers/Home/index.tsx [hunk 1]
```diff
@@ -1,16 +1,101 @@
...
+  console.log("numberRef", numberRef.current);
+
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Welcome to Faba onboarding project - Job board
-        </p>
+        {/* <img src={logo} className="App-logo" alt="logo" /> */}
```
Comment (tiencmavixo) [inline src/containers/Home/index.tsx:63]: remove, when don't use

File: src/containers/Home/index.tsx [hunk 2]
```diff
@@ -1,16 +1,101 @@
...
+          {/* <ul>
+            {job.map((j) => {
+              return (
+                <li key={j.id} style={{ color: "yellow", fontSize: "12px" }}>
+                  {j.name}
+                  <span style={{ color: "green" }}> {j.status}</span>
+                </li>
+              );
+            })}
+          </ul> */}
```
Comment (tiencmavixo) [inline src/containers/Home/index.tsx:125]: remove, when don't use

File: src/containers/Home/index.tsx [hunk 3]
```diff
@@ -1,16 +1,101 @@
...
+      setNumber(numberRef.current);
+    }
+    setNumber(number - 10);
+    if (number < 1) {
+      setNumber(150);
+      numberRef.current = 141;
+    }
+  }
+  console.log("number", number);
+  console.log("numberRef", numberRef.current);
```
Comment (tiencmavixo) [inline src/containers/Home/index.tsx:58]: remove console.log

File: src/containers/Home/index.tsx [hunk 4]
```diff
@@ -1,16 +1,101 @@
...
 import "./styles.scss";
-
+interface Job {
+  id: number;
+  name: string;
+  status: string;
+}
+interface Jobs {
+  job: Job[];
+}
```
Comment (tiencmavixo) [inline src/containers/Home/index.tsx:12]: don't need theme

File: src/containers/Home/index.tsx [hunk 5]
```diff
@@ -1,16 +1,129 @@
...
+    }
+    setNumber(number - 10);
+    if (number < 1) {
+      setNumber(150);
+      numberRef.current = 141;
+    }
+  }
+  function changeFindName(event: React.ChangeEvent<HTMLInputElement>) {
+    setJobName(event.target.value);
+    console.log(jobName);
```
Comment (tiencmavixo) [inline src/containers/Home/index.tsx:52]: remove console.log

**Comment by tiencmavixo** [comment]
please, remove code which not use

File: src/containers/Home/styles.scss [hunk 6]
```diff
@@ -32,3 +34,28 @@
     transform: rotate(360deg);
   }
 }
+.productList{
+  display:flex;
+  flex-basis: 25%;
+  flex-direction: row;
+  flex-wrap: wrap;
+  .box{
```
Comment (tiencmavixo) [inline src/containers/Home/styles.scss:42]: please, format by BEM rule

File: src/containers/Home/styles.scss [hunk 7]
```diff
@@ -32,3 +34,28 @@
...
+      background-color: #fff;
+      border-radius: 30px;
+      flex-direction: column;
+      justify-content: center;
+      h3{
+
+      }
+      h4{
+
+      }
```
Comment (tiencmavixo) [inline src/containers/Home/styles.scss:58]: remove h3, h4

Comment (vutiendanhvgu01) [reply src/containers/Home/styles.scss]: em fix rồi

File: src/containers/Home/index.tsx (hunk 5 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: em fix rồi

File: src/containers/Home/index.tsx (hunk 4 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: em fix rồi

File: src/containers/Home/index.tsx (hunk 3 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: em fix rồi

File: src/containers/Home/index.tsx (hunk 1 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: em fix rồi

File: src/containers/Home/index.tsx (hunk 2 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: em fix rồi

File: src/containers/Home/styles.scss (hunk 7 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/styles.scss]: I have already fixed

File: src/containers/Home/index.tsx (hunk 5 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: I have already fixed

File: src/containers/Home/index.tsx (hunk 4 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: I have already fixed

File: src/containers/Home/index.tsx (hunk 3 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: I have already fixed

File: src/containers/Home/index.tsx (hunk 2 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: I have already fixed

File: src/containers/Home/index.tsx (hunk 1 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: I have already fixed

File: src/containers/Home/index.tsx [hunk 8]
```diff
@@ -1,17 +1,46 @@
...
+}
 
 function Home() {
+  const [job, setJob] = useState<Job[]>([]);
+  const [count, setCount] = useState(0);
+
+  const getJobAPi = async () => {
+    try {
+      const response = await axios.get(
+        "https://640ef43f4ed25579dc412e27.mockapi.io/api/showJob/job"
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:18]: https://640ef43f4ed25579dc412e27.mockapi.io/api should to store at env

File: src/containers/Home/index.tsx [hunk 9]
```diff
@@ -1,17 +1,46 @@
...
+      );
+      setJob(response.data);
+    } catch (erro) {
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:21]: typo

Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: I have already fixed

File: src/index.tsx [hunk 10]
```diff
@@ -1,19 +1,25 @@
...
+import { Provider } from "react-redux";
+dotenv.config();
 ReactDOM.render(
   <React.StrictMode>
-    <BrowserRouter>
-      <App />
-    </BrowserRouter>
+    <Provider store={store}>
+      <BrowserRouter>
+        <Home></Home>
```
Comment (Tiencmfaba) [inline src/index.tsx:18]: why don't use <Home />

File: src/redux/reducer/JobReducer.tsx [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+      state.jobs = action.payload;
+    },
+  },
+});
+
+export const { getJobAction } = JobReducer.actions;
+
+export const jobAPI = () => {
+  return async (dispatch: DispatchType) => {
+    const result = await axios({
```
Comment (Tiencmfaba) [inline src/redux/reducer/JobReducer.tsx:26]: why don't use axios.get

File: src/redux/reducer/JobReducer.tsx [hunk 12]
```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import axios from "axios";
+import { Job } from "../../containers/Home";
+import { DispatchType } from "../configStore";
+export interface JobState {
+  jobs: Job[];
+}
+const initialState = {
```
Comment (Tiencmfaba) [inline src/redux/reducer/JobReducer.tsx:8]: add type for initialState

File: src/index.tsx (hunk 10 above)
Comment (Tiencmfaba) [reply src/index.tsx:18]: `<Home />`

Comment (vutiendanhvgu01) [reply src/index.tsx]: I have already fixed

File: src/redux/reducer/JobReducer.tsx (hunk 11 above)
Comment (vutiendanhvgu01) [reply src/redux/reducer/JobReducer.tsx]: I have already fixed

File: src/redux/reducer/JobReducer.tsx (hunk 12 above)
Comment (vutiendanhvgu01) [reply src/redux/reducer/JobReducer.tsx]: I have already fixed

**Pull request merged**